src/main/java/net/fabricmc/loader/api/metadata/CustomValue.java

```diff
@@ -67,6 +71,23 @@
 	 */
 	boolean getAsBoolean();
 
+	/**
+	 * Compares this custom value with another and returns whether they are equal.
+	 *
+	 * @param another The custom value to compare to.
+	 */
+	boolean equals(Object another);
+
+	/**
+	 * Returns the hash code for this custom value.
+	 */
+	int hashCode();
+
+	/**
+	 * Returns the value of this custom value as json.
+	 */
+	String toString();
```

Comment:
Do we really need to define these explicitly here? These are all defined in Object which all interfaces implicitly are implemented by. My guess is it makes the api more obvious in support.